src/components/Modal/ModalFrame.jsx

```diff
@@ -3,11 +3,16 @@ import CloseIcon from "../Icon/CloseIcon";
 import Button from "../UI/Button";
 import PropTypes from "prop-types";
 
-const ModalFrame = ({ modalType, children }) => {
+const ModalFrame = ({ isClear, isExistCloseButton, modalType, children }) => {
```

Comment:
```suggestion
const ModalFrame = ({ modalType, isClear, hasCloseButton, children }) => {
```
닫기 버튼이 있는지 여부에 따라 Bool 값을 주기 위해 `has`를 활용해볼 수 있을 것 같습니다!
`isExistCloseButton`의 경우 '기존에 존재했던 닫기 버튼'으로 인지할 수 있는 부분이 있다고 생각했습니다. 검토 부탁 드립니다!

Reply:
`hasCloseButton`으로 수정 완료하였습니다 :)